docs/development_environment.md

```diff
@@ -41,6 +41,10 @@ When a task is currently running (like `Preview` for the docs), it can be restar
 
 It is also possible to set up a more traditional development environment. See the section for your operating system.
 
+:::tip
```

Comment:
Tips get a lot of space and attract a lot of attention. I think that we can put it more subtle as part of the paragraph above.